### DISCUSSION TITLE
Order parameters CV

### DISCUSSION THREAD

**Discussion by jok16** [SSAGESLabs/PySAGES#251] [Q&A]
Are order parameters such as nematic or Steinhardt order parameters implemented as CVs ? Or is there a way to implement them as colvars ? Thanks

**Comment by p-zubieta** [comment]
Not officially, yet. I do have a question though, are you thinking on using a backend that supports orientations or can they be computed from the positions of the particles?

If the answer to this is yes to the latter, then it should not be too hard to write a custom implementation, for the first case, it would still be possible to implement this, but that would require some changes to the backend code first.
